### PR TITLE
Removing Objective-C code

### PR DESCRIPTION
**Pull request by magic146**

### CODE REVIEW

File: ChattoAdditions/Source/UI Components/CircleProgressIndicatorView/CircleIconView.swift [hunk 1]
```diff
@@ -0,0 +1,271 @@
+//
```
Comment (AntonPalich) [inline ChattoAdditions/Source/UI Components/CircleProgressIndicatorView/CircleIconView.swift]: Wrong header

File: ChattoAdditions/Source/UI Components/CircleProgressIndicatorView/CircleIconView.swift [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+//  CircleIconView.swift
+//  ChattoAdditions
+//
+//  Created by Mikhail Gasanov on 29/05/2019.
+//  Copyright © 2019 Badoo. All rights reserved.
+//
+
+import UIKit
+
+public enum CircleIconType {
```
Comment (AntonPalich) [inline ChattoAdditions/Source/UI Components/CircleProgressIndicatorView/CircleIconView.swift]: It's a great opportunity to clean up this component and remove all unused code. We don't use the most of its functionality.

File: ChattoAdditions/Source/UI Components/CircleProgressIndicatorView/CircleIconView.swift (hunk 1 above)
Comment (magic146) [reply ChattoAdditions/Source/UI Components/CircleProgressIndicatorView/CircleIconView.swift]: Fixed.

File: ChattoAdditions/Source/UI Components/CircleProgressIndicatorView/CircleIconView.swift (hunk 2 above)
Comment (magic146) [reply ChattoAdditions/Source/UI Components/CircleProgressIndicatorView/CircleIconView.swift]: After a discussion, we decided to leave everything as it is.

**Pull request merged**